tests/test_notebooks.py

```diff
@@ -43,6 +45,22 @@ def test_eln_import(selenium_driver):
     driver.find_element(By.ID, "tooltip")
 
 
-def test_computational_resources(selenium_driver):
-    driver = selenium_driver("notebooks/computational_resources.ipynb")
+def test_computational_resources(notebook_service, selenium):
+    """Test the quicksetup of the code"""
+    url, token, docker_compose = notebook_service
+
+    # install the aiida-quantumespresso, we need it for test of some widgets
+    install_command = "bash -c 'pip install -U aiida-quantumespresso==4.1.0'"
+    command = f"exec --workdir /home/jovyan/apps/aiidalab-widgets-base -T aiidalab {install_command}"
+
+    docker_compose.execute(command)
+
+    url_with_token = urljoin(
+        url,
+        f"apps/apps/aiidalab-widgets-base/notebooks/computational_resources.ipynb?token={token}",
+    )
+    selenium.get(f"{url_with_token}")
+    selenium.implicitly_wait(10)  # must wait until the app loaded
```

Comment:
This should be set in the `notebook_service` fixture in `conftest`, that way it will be applied to all tests. Same way as it is done in the QeApp:

https://github.com/aiidalab/aiidalab-qe/blob/50406cc3cf346d9dcfd123ddb5e7f0b345a0c313/tests/conftest.py#L55

Reply:
This is not true, the scope of `selenium` fixture is the "function", it will create for every test. Check here: https://github.com/pytest-dev/pytest-selenium/blob/47f4ef38f4c479c7fab0b9644043cafa9564edeb/src/pytest_selenium/pytest_selenium.py#L223-L225

Reply:
I am not sure I understand your point, but yes I meant to modify the `selenium_drive` fixture, sorry.